src/Search_Replace_Command.php

```diff
@@ -269,8 +275,16 @@ public function __invoke( $args, $assoc_args ) {
 
 		// Get table names based on leftover $args or supplied $assoc_args
 		$tables = \WP_CLI\Utils\wp_get_table_names( $args, $assoc_args );
+
+		// Removes from $tables tables provided by skip-tables argument
+		$tables = array_diff( $tables, $this->skip_tables );
+
 		foreach ( $tables as $table ) {
 
+			if ( in_array( $table, $this->skip_tables ) ) {
```

Comment:
Just to note that this wasn't removed when the `array_diff()` was added and made it redundant.

Reply:
Good spot, #50